test/integration/test/captions.test.js

```diff
@@ -1,9 +1,33 @@
 const clientHelper = require('../src/client-helper');
 const configHelper = require('../src/config-helper');
+const JobStatus = require('../../../dist/src/models/JobStatus').JobStatus;
+const client = clientHelper.getClient(configHelper.getApiKey());
 const InstanceStateError = require('../../../src/models/RevAiApiError').InvalidStateError;
 
+beforeAll(async (done) => {
+    const jobList = await client.getListOfJobs();
+    var jobId;
+    if(jobList != undefined) {
+        jobId = clientHelper.getTranscribedJobId(jobList);
+    }
+    if(jobId == undefined) {
+        const job = await client.submitJobUrl('https://www.rev.ai/FTC_Sample_1.mp3');
+        jobId = job.id;
+    }
+    var count = 0;
+    var intervalObject = setInterval(function(){ 
+        count++;
+        (async () => {
+            const jobDetails = await client.getJobDetails(jobId);
+            if (jobDetails.status == JobStatus.Transcribed || count >= 40) { 
```

Comment:
What is the purpose of the count? Won't beforeAll timeout before we get to it?

Reply:
Yea you are right. I'll remove the count.